docs/source/conf.py

```diff
@@ -84,7 +92,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "theme"
+html_theme = "sphinx_book_theme"
 html_theme_path = ["."]
```

Comment:
I think this line can be removed and the `cyclops/docs/source/theme` directory can be removed once the new theme kicks in.